src/ghQueueMicrotask.ts

```diff
@@ -0,0 +1,2 @@
+export const ghQueueMicrotask =
+  typeof queueMicrotask === 'function' ? queueMicrotask : setImmediate;
```

Comment:
Add a comment explaining why it's there and we're golden.

Reply:
Sure, done in [795f35e](https://github.com/software-mansion/react-native-gesture-handler/pull/2612/commits/795f35ec703a22cdcd5adad34763fbd713fc7eb6)